packages/devtools_testing/lib/service_manager_test.dart

```diff
@@ -201,17 +201,15 @@ Future<void> runServiceManagerTests(FlutterTestEnvironment env) async {
       await env.tearDownEnvironment();
     });
 
-    // TODO(kenz): once hot restart tests are fixed, add a hot restart test
-    // that verifies the state of service extensions after a hot restart.
-    // TODO(jacobr): uncomment out the hotRestart tests once
+    // TODO(kenz): add a hot restart test that verifies the state of service
+    // extensions after a hot restart.
     // https://github.com/flutter/devtools/issues/337 is fixed.
```

Comment:
looks like something happened here with the comment? the bug references a todo line that was deleted

Reply:
cleaned up.